assets/rollup.config.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Rollup configuration.
+ * NOTE: This is a CommonJS module so it can be imported by Karma.
+ */
+
+const buble = require('rollup-plugin-buble');
```

Comment:
Not a Javascript expert, so I'm going ask the question: why can't you use ES6's import syntax here?

Reply:
I wanted to import from the Karma config... since that's run under Node this one needed to be CommonJS also.